vistas/usuarios/crear.php

```diff
@@ -0,0 +1,8 @@
+<main class="contenedor seccion">
+  <h1>Crear Usuario(a)</h1>
```

Comment:
```suggestion
  <h1>Crear Usuario</h1>
```
Creo que no es necesario agregarlo ya que usuario es un objeto como tal